client/galaxy/scripts/mvc/toolshed/categories-view.js

```diff
@@ -60,44 +55,44 @@ var ToolShedCategories = Backbone.View.extend({
         });
     },
 
-    templateCategoryList: _.template(
-        [
-            '<style type="text/css">',
-            ".ui-autocomplete { background-color: #fff; }",
-            "li.ui-menu-item { list-style-type: none; }",
-            "</style>",
-            '<div class="unified-panel-header" id="panel_header" unselectable="on">',
-            '<div class="unified-panel-header-inner" style="layout: inline;">Categories in <%= tool_shed.replace(/%2f/g, "/") %><a class="ml-auto" href="#/queue">Repository Queue (<%= queue %>)</a></div>',
-            "</div>",
-            '<div class="unified-panel-body" id="list_categories">',
-            '<div id="standard-search" style="height: 2em; margin: 1em;">',
-            '<span class="ui-widget" >',
-            '<input class="search-box-input" id="search_box" data-shedurl="<%= tool_shed.replace(/%2f/g, "/") %>" name="search" placeholder="Search repositories by name or id" size="60" type="text" />',
-            "</span>",
-            "</div>",
-            '<div style="clear: both; margin-top: 1em;">',
-            '<table class="grid">',
-            '<thead id="grid-table-header">',
-            "<tr>",
-            "<th>Name</th>",
-            "<th>Description</th>",
-            "<th>Repositories</th>",
-            "</tr>",
-            "</thead>",
-            "<% _.each(categories, function(category) { %>",
-            "<tr>",
-            "<td>",
-            '<a href="#/category/s/<%= tool_shed %>/c/<%= category.get("id") %>/k/name/p/1/t/asc"><%= category.get("name") %></a>',
-            "</td>",
-            '<td><%= category.get("description") %></td>',
-            '<td><%= category.get("repositories") %></td>',
-            "</tr>",
-            "<% }); %>",
-            "</table>",
-            "</div>",
-            "</div>"
-        ].join("")
-    )
+    templateCategoryList: function() {
```

Comment:
Can we use multi-line strings here?

Reply:
converted all the templates to multiline, thanks!